src/pgo.app.src

```diff
@@ -7,7 +7,6 @@
    [kernel,
     stdlib,
     backoff,
-    opentelemetry_api,
```

Comment:
On OSX, I get a TON of failures with this set. A bunch of `already_started` errors with OTEL. Unsure why I didn't see this on my Linux machine but @tsloughter did make a note it was weird.

Reply:
Hm, it is a runtime dependency so needs to be there. Can you add it back and I'll try to reproduce as well.

Reply:
ac0a6627d29e8587a7a4ef5b1302f3bf40bb7ece